File: app/src/main/java/com/patapin/djmojo/soundbox/activity/StatActivity.java
package com.patapin.djmojo.soundbox.activity;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

import com.patapin.djmojo.soundbox.R;
import com.patapin.djmojo.soundbox.model.ButtonInfos;

import java.util.List;

public class StatActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_stat);

        List<ButtonInfos> mainSoundList = MainActivity.getMainSoundList();
        List<ButtonInfos> moviesSoundList = MainActivity.getMoviesSoundList();
        List<ButtonInfos> southparkSoundList = MainActivity.getSouthParkSoundList();

        for (ButtonInfos buttonInfos : mainSoundList) {

            TextView textView = (TextView) findViewById(buttonInfos.getStatTextId());
            String text = String.valueOf(buttonInfos.getClick().getCounter());
            textView.setText(text);
        }

        for (ButtonInfos buttonInfos : moviesSoundList) {

            TextView textView = (TextView) findViewById(buttonInfos.getStatTextId());
            String text = String.valueOf(buttonInfos.getClick().getCounter());
            textView.setText(text);
        }

        for (ButtonInfos buttonInfos : southparkSoundList) {

            TextView textView = (TextView) findViewById(buttonInfos.getStatTextId());
            String text = String.valueOf(buttonInfos.getClick().getCounter());
            textView.setText(text);
        }

    }
}

File: app/src/main/java/com/patapin/djmojo/soundbox/activity/StatActivity.kt
package com.patapin.djmojo.soundbox.activity

import android.app.Activity
import android.os.Bundle
import android.widget.TextView
import com.patapin.djmojo.soundbox.R

class StatActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_stat)

        val mainSoundList = MainActivity.getMainSoundList()
        val moviesSoundList = MainActivity.getMoviesSoundList()
        val southparkSoundList = MainActivity.getSouthParkSoundList()

        mainSoundList?.forEach { buttonInfos ->
            val textView = findViewById<TextView>(buttonInfos.statTextId)
            val text = buttonInfos.click!!.counter.toString()
            textView.text = text
        }

        moviesSoundList?.forEach { buttonInfos ->
            val textView = findViewById<TextView>(buttonInfos.statTextId)
            val text = buttonInfos.click!!.counter.toString()
            textView.text = text
        }

        southparkSoundList?.forEach { buttonInfos ->
            val textView = findViewById<TextView>(buttonInfos.statTextId)
            val text = buttonInfos.click!!.counter.toString()
            textView.text = text
        }

    }
}
